src/parse.rs

```diff
@@ -41,6 +42,7 @@ pub fn parse(
         ("BigInt", "int64"),
         ("Numeric", "string"),
         ("Text", "string"),
+        ("Date", "NaiveDate"),
```

Comment:
Date is not in protobuf.

Reply:
We could use string or int64, from the [primitive types](https://developers.google.com/protocol-buffers/docs/proto3#scalar), or use [timestamp](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#timestamp) ?

Reply:
Both could work, but we will need extra conversion code for from and to functions, so just adding a conversion pair here does not seem working.
Please suggest yours if you have preferred conversion practice.